src/main/kotlin/no/nav/medlemskap/regler/funksjoner/ArbeidsforholdFunksjoner.kt

```diff
@@ -34,6 +34,11 @@ object ArbeidsforholdFunksjoner {
     infix fun List<Arbeidsforhold>.antallAnsatteHosArbeidsgivere(kontrollPeriode: Kontrollperiode): List<Int> =
         ansatteHosArbeidsgivere(kontrollPeriode).map { it.antall ?: 0 }
 
+    infix fun List<Arbeidsforhold>.antallAnsatteHosArbeidsgiversJuridiskeEnheter(kontrollPeriode: Kontrollperiode): List<Int?> =
+        arbeidsforholdForKontrollPeriodeMedStillingsprosentOver0(kontrollPeriode).filter {
+            !it.arbeidsgiver.juridiskeEnheter.isNullOrEmpty()
```

Comment:
Hadde det vært mulig å skrive it.arbeidsgiver.juridiskeEnheter.IsNotNullOrEmpty() ?

Reply:
Tenkte faktisk på det samme, men den funksjonen finnes ikke! Men siden du nevner kan jeg lage en extension-funksjon på det. Den funksjonen tenker jeg kan brukes på alle collections.

Reply:
Åja, det stemmer jo. Merkelig at det ikke finnes siden funksjonen for å sjekke på om det er null eller empty finnes. Skulle trodd at å sjekke det motsatte ville vært populært også....

Bare å kjøre på med en extension hvis du vil det!